skins/oasis/modules/BodyController.class.php

```diff
@@ -161,7 +161,7 @@ public function getRailModuleList() {
 			$railModuleList = [
 				1202 => [ 'Forum', 'forumRelatedThreads', null ],
 				1201 => [ 'Forum', 'forumActivityModule', null ],
-				1490 => [ 'Ad', 'Index', [ 'slotName' => 'TOP_RIGHT_BOXAD' ] ],
+				1490 => [ 'Ad', 'Index', [ 'slotName' => 'TOP_BOXAD' ] ],
```

Comment:
Cache is going to hurt us 🙈

Reply:
Inject both slots, for a while. It shouldn't hurt us. I believe

Reply:
Fixed.